sdk/firstload/src/index.tsx

```diff
@@ -75,9 +79,26 @@ const H: HighlightPublicInterface = {
 				return
 			}
 
+			let previousSession = getPreviousSessionData()
+			let sessionSecureID = GenerateSecureID()
+			if (previousSession?.sessionSecureID) {
+				sessionSecureID = previousSession.sessionSecureID
+			} else {
+				const sessionData: SessionData = {
+					...previousSession,
+					projectID: +projectID,
```

Comment:
i'm going to use this trick more :)

Reply:
There are so many of these stupid tricks. For example, you can use `!!~someArrayIndex` to test if `someArrayIndex` is `-1`. The `~` sign is the bitwise NOT operator that just so happens to shift `-1` to `0`. It's typically used to test if `Array.findIndex` was successful.

![image](https://github.com/highlight/highlight/assets/878947/4c963488-0fad-4706-9b2c-38b447ff84ee)